app/geo/gui/src/ts/twxplore/gui/geo/featureTypeStrategies/FeatureTypeStrategy.ts

```diff
@@ -1,10 +1,22 @@
 import {FeatureType, FeatureQuery} from "../api/graphqlGlobalTypes";
 import {FeatureAttributeName} from "../states/map/FeatureAttributeName";
+import {Dispatch} from "redux";
+import {FeaturesByType} from "../states/map/FeaturesByType";
 
 export interface FeatureTypeStrategy {
   readonly name: FeatureType;
   readonly childFeatureTypes: FeatureType[];
   readonly expandable: boolean;
   readonly fieldsToShowOnPopup: FeatureAttributeName[];
   getClickedQuery(clickedFeatureUri: string): FeatureQuery | null;
+  dispatchLayerConfigurationActions(
+    keplerLayerOfFeatureType: any,
```

Comment:
kwds: { ... }

Reply:
The other nice thing about kwds is that you can put them in an interface and reuse them.
kwds: LayerConfigurationActionParameters
interface LayerConfigurationActionParameters { ... }
This is called a "parameter object": https://wiki.c2.com/?ParameterObject